.github/workflows/package-release.yml

```diff
@@ -8,10 +8,24 @@ on:
 
 jobs:
   build-pecl:
+    environment: release
```

Comment:
The environment safeguards credentials as they won't be available to other CI jobs. In addition to that, we can add extra checks to releases, e.g. requirements for reviews, time delays, etc.